modules/nf-core/md5sum/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+nextflow_process {
+
+    name "Test Process MD5SUM"
+    script "../main.nf"
+    process "MD5SUM"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "md5sum"
+
+    test("md5sum on hello.txt") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
```

Comment:
is the Channel.of part needed here and on the next test? Never seen it in nf-tests